src/block-components/separator/style.scss

```diff
@@ -2,8 +2,9 @@
 
 .stk-block {
 	&:is(.stk-has-top-separator, .stk-has-bottom-separator) {
-		> *:not(.stk-video-background) { // Don't change the order of any video backgrounds, they stay at the back.
+		> *:not(.stk-video-background, .stk-separator) { // Don't change the order of any video backgrounds, they stay at the back.
 			z-index: 2;
+			position: relative;
```

Comment:
can we not place a `position: relative` here because this might cause unwanted side effects.